modules/nw-network-observability-operator.adoc

```diff
@@ -0,0 +1,79 @@
+/ Module included in the following assemblies:
+// * networking/network_observability/understanding-network-observability-operator.adoc
+
+:_content-type: PROCEDURE
+[id="nw-network-observability-operator_{context}"]
+= Network Observability Operator
+
+When you deploy flow collector is deployed, the Operator provides dashboards, metrics, and keeps flows accessible in a queryable log store, Grafana Loki. When a FlowCollector is deployed, new dashboards are available in the Console.
```

Comment:
This hurts my head.  What are we trying to convey here? 
I'm not sure but I think we are trying to talk about the flow collector?
I don't know why we would say anything about a queryable log store.  
I also don't know why we would talk about deploying flowcollector and a new dashboard appears.

I think we want to talk about the overall network observability installation - and part of that is deploying a flow collector.  Upon successful install and configuration of the flow collector, a new network section is available in the ocp console's observe section.